src/instantx.cpp

```diff
@@ -312,22 +311,27 @@ void CreateTxLockVote(CTransaction& tx, int64_t nBlockHeight)
 //received a consensus vote
 bool ProcessTxLockVote(CNode* pnode, CTxLockVote& vote)
 {
-    int n = mnodeman.GetMasternodeRank(vote.vinMasternode, vote.nBlockHeight, MIN_INSTANTSEND_PROTO_VERSION);
+
+    LogPrint("instantsend", "ProcessTxLockVote -- Transaction Lock Vote, txid=%s\n", vote.txHash.ToString());
 
     CMasternode* pmn = mnodeman.Find(vote.vinMasternode);
-    if(pmn != NULL)
-        LogPrint("instantsend", "ProcessTxLockVote -- Masternode addr=%s, rank: %d\n", pmn->addr.ToString(), n);
+    if(pmn == NULL) {
```

Comment:
Should change this non thread-safe call to use GetMasternodeInfo(), or simply Has(), if this is just to determine existence of the masternode